x-pack/plugins/security_solution/server/endpoint/routes/metadata/index.ts

```diff
@@ -106,4 +107,23 @@ export function registerEndpointRoutes(
     },
     getMetadataRequestHandler(endpointAppContext, logger)
   );
+
+  router.get(
+    {
+      path: `/api/endpoint/toAgentID`,
+      validate: { query: schema.object({ ids: schema.string() }) },
+    },
+    async (context, req, res) => {
+      const agents = await GetAgentIDsForEndpoints(
```

Comment:
Its strange to have a function's name starting in upper case - at first (when I saw it in the `import`) I thought it was a Type or a Class. I suggest making it lower case to be consistent